client/base_client.go

```diff
@@ -55,10 +61,12 @@ type baseClient struct {
 
 	security SecurityOption
 
-	gRPCDialOptions  []grpc.DialOption
-	timeout          time.Duration
-	maxRetryTimes    int
-	enableForwarding bool
+	gRPCDialOptions         []grpc.DialOption
+	timeout                 time.Duration
+	maxRetryTimes           int
+	enableForwarding        bool
+	maxTSOBatchSize         int
+	maxTSOBatchWaitInterval time.Duration
```

Comment:
Do they need to be placed in `baseClient`?

Reply:
I will file another PR to refactor this part.

Reply:
PTAL https://github.com/tikv/pd/pull/4104.